### PR TITLE
buid(deps): bump Yarn from 3.5.0 to 3.6.0

### PR DESCRIPTION
**Pull request by yeikel**
Changelog: https://github.com/yarnpkg/berry/compare/@yarnpkg/cli/3.5.0...@yarnpkg/cli/3.6.0

### CODE REVIEW

**Review by jurre — APPROVED** [review]
We'll need to hold off merging this for a bit more, but going to approve so we know that it's good to go once we're ready

**Comment by yeikel** [comment]
> We'll need to hold off merging this for a bit more, but going to approve so we know that it's good to go once we're ready

Please note that 3.6.0 is out and I just updated this pull request

**Review by deivid-rodriguez — APPROVED** [review]
It's hard to review this upgrades since there's no upstream changelog, but I'd say let's do this and trust our tests. The less unreleased changes we accumulate, the easier will be to figure out regressions.

Also, to be noted that since we use corepack, we're already running Yarn versions higher than 3.5.0 after all, for users that are making that explicit through their `package.json` file.

**Pull request merged**
